nautobot_dev_example/models.py

```diff
@@ -4,17 +4,18 @@
 from django.db import models
 
 # Nautobot imports
-from nautobot.apps.models import PrimaryModel
+from nautobot.apps.models import PrimaryModel, extras_features
 
 # from nautobot.apps.models import extras_features
 # If you want to use the extras_features decorator please reference the following documentation
 # https://docs.nautobot.com/projects/core/en/latest/plugins/development/#using-the-extras_features-decorator-for-graphql
 # Then based on your reading you may decide to put the following decorator before the declaration of your class
-# @extras_features("custom_fields", "custom_validators", "relationships", "graphql")
+# @extras_features("custom_links", "custom_validators", "export_templates" "graphql", "webhooks")
```

Comment:
Remove?

Reply:
I left it as the line directly above references it.

Reply:
This whole section can go. It's duplicated below